components/image-releaser/app.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: 0.1.0
+environments:
+  default:
+    destination:
+      namespace: default
+      server: https://35.226.49.107
+    k8sVersion: v1.8.8
+    path: default
+  releasing:
+    destination:
+      namespace: default
+      server: https://35.226.49.107
+    k8sVersion: v1.7.0
+    path: releasing
+  testing:
```

Comment:
Since this is intended for releasing, can we strip this down to a single environment?